src/PeriodEnd/PaymentsDue/SFA.DAS.ProviderPayments.Calc.PaymentsDue/Domain/MatchSetForPayments.cs

```diff
@@ -84,5 +64,54 @@ public override int GetHashCode()
                 return hash;
             }
         }
+
+        public static bool operator ==(MatchSetForPayments left, MatchSetForPayments right)
```

Comment:
What is the purpose of MatchSetPayments? I thought we were just adding unit tests